app/src/main/java/com/hillnerds/soundsparrow/ChannelSoundGeneration.java

```diff
@@ -0,0 +1,177 @@
+package com.hillnerds.soundsparrow;
+
+import android.util.Log;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.Random;
+
+/**
+ * Created by aga on 06/02/17.
+ */
+
+public class ChannelSoundGeneration {
+
+    int scale;
```

Comment:
Every function in this class is static, are these variables actually used anywhere? If so, consider just using them in the function that uses them instead of declaring them non-static out here.